easybuild/easyconfigs/q/qforce/qforce-0.6.11-foss-2022a.eb

```diff
@@ -26,7 +26,9 @@ use_pip = True
 
 exts_list = [
     ('PuLP', '2.1', {
-        'checksums': ['76f9c9f984e6e407959545a556ce10c6cc0fdb0db820d627f0073ecf4f5c5c1b'],
+        'source_urls': ['https://github.com/coin-or/pulp/archive'],
+        'sources': ['v%(version)s.tar.gz'],
+        'checksums': ['1436f33239676206c34498047b0e339ec529c898daace75a9b17a5fd70d21fbe'],
```

Comment:
This changes the PuLP download from the file hosted at pypi - available at https://pypi.org/project/PuLP/2.1/#files - to the one from GitHub - available at https://github.com/coin-or/pulp/releases/tag/v2.1

`pulp-2.1` corresponds to the GitHub download and `PuLP-2.1` to the pypi one.

```diff
$ diff -r pulp-2.1/ PuLP-2.1
Only in pulp-2.1/: .coin-or
Only in pulp-2.1/: doc
Only in pulp-2.1/examples: BeerDistributionProblemCompetitorExtension
Only in pulp-2.1/examples: BeerDistributionProblemWarehouseExtension
Only in pulp-2.1/: .github
Only in pulp-2.1/: .gitignore
Only in PuLP-2.1: PKG-INFO
Only in pulp-2.1/pulp/solverdir/cbc: __init__.py
Only in pulp-2.1/pulp/solverdir/cbc/linux/32: cbc
Only in pulp-2.1/pulp/solverdir/cbc/linux/32: coin-license.txt
Only in pulp-2.1/pulp/solverdir/cbc/linux/64: cbc
Only in pulp-2.1/pulp/solverdir/cbc/linux/64: coin-license.txt
Only in pulp-2.1/pulp/solverdir/cbc/linux: __init__.py
Only in pulp-2.1/pulp/solverdir/cbc/osx/64: cbc
Only in pulp-2.1/pulp/solverdir/cbc/osx/64: coin-license.txt
Only in pulp-2.1/pulp/solverdir/cbc/osx: __init__.py
Only in pulp-2.1/pulp/solverdir/cbc/win/32: cbc.exe
Only in pulp-2.1/pulp/solverdir/cbc/win/32: coin-license.txt
Only in pulp-2.1/pulp/solverdir/cbc/win/64: cbc.exe
Only in pulp-2.1/pulp/solverdir/cbc/win/64: coin-license.txt
Only in pulp-2.1/pulp/solverdir/cbc/win: __init__.py
Only in pulp-2.1/pulp/solverdir/choco: choco-parsers-with-dependencies.jar
Only in pulp-2.1/pulp/solverdir/choco: license.txt
Only in PuLP-2.1: PuLP.egg-info
Only in pulp-2.1/: ROADMAP
Only in PuLP-2.1: setup.cfg
Only in pulp-2.1/: .travis.yml
```

Can you clarify what issue changing the source solves? Is some of the above required for PuLP to function correctly?

Reply:
The reason was that PuLP needs a solver in general and PuLP-2.1 from pypi doesn't have the default solver Cbc, while the github download does. But as a matter of fact, a better way to include it is to add it as a dependency.